src/html/chat.html

```diff
@@ -33,7 +33,43 @@
 {% endblock %}
 
 {% block content %}
-<div id="chat-wrapper">
-    <iframe id="chat-frame" src="https://kiwiirc.com/client/irc.esper.net:+6697/?nick=sponge|?#sponge"></iframe>
+<header>
+    <div class="container">
+        <div class="logo">
+            <img src="assets/img/icons/spongie-mark-dark.svg" alt="">
```

Comment:
Same here, use the alt tag or remove it. People that use screen readers will thank you.